packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4617,6 +4618,66 @@ QUnit.module('Aria accessibility', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('aria-label should be \'Read-only calendar\' and role = group when readOnly option is true', function(assert) {
```

Comment:
it's better to use double quotes for the nested string because it simplified a search using ctr-f copying report name from a test page